### PR TITLE
fix(nuxt): autoImport exclude `.nuxt`

### PR DESCRIPTION
**Pull request by cinob**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

It has caused nuxt3 to take a very long time to build since I upgraded from rc4 to rc6

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by antfu** [comment]
I would like to know what this is trying to fix first

**Comment by cinob** [comment]
autoimport scans all the files in the .nuxt folder generated during the build process, causing the larger the project the longer it takes to build.
My current project server building has taken more than 9 minutes

**Comment by cinob** [comment]
Are the files in the .nuxt folder skippable during the nitro build server?

File: packages/nuxt/src/core/nitro.ts [hunk 1]
```diff
@@ -99,6 +99,9 @@ export async function initNitro (nuxt: Nuxt) {
     },
     rollupConfig: {
       plugins: []
+    },
+    autoImport: {
+      exclude: [/[\\/]node_modules[\\/]/, /[\\/]\.git[\\/]/, /[\\/]\.nuxt[\\/]/]
```
Comment (antfu) [inline packages/nuxt/src/core/nitro.ts:104]: We might also need `.output` and `dist`?

**Comment by pi0** [comment]
Thanks so much for your pull request @cinob. As @danielroe mentioned, this is probably an enhancement we can do in upstream nitro package to fix. Here is the PR: https://github.com/unjs/nitro/pull/355

Hope see more contributions from you and cheers!

**Pull request closed without merging**